doc/examples/Finite_State_Projection/utils.py

```diff
@@ -1,3 +1,4 @@
+"""Utility functions, that simulate and plot the FSP example."""
```

Comment:
```suggestion
"""Utility functions which simulate and plot the FSP example."""
```

Reply:
Still should be `that` without comma (`which` with comma)

Reply:
true, you win :rofl: